config/default/aad-pod-identity-deployment.yaml

```diff
@@ -63,7 +63,7 @@ metadata:
     k8s-app: capz-aad-pod-id-nmi-binding
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: manager
```

Comment:
The account that is set here I believe it should also be set in aad-pod-identity NMI DaemonSet `.Spec.Template.Spec.ServiceAccountName` field (in this same file), which is currently set to `default`.

Reply:
Set it to `manager`. PTAL.